docs/whats-news/2023/2023-q2.mdx

```diff
@@ -5,7 +5,92 @@ slug: /whats-news/2023-Q2
+
+🍏🤖 [First-run Release Date](https://mozilla-hub.atlassian.net/browse/EXP-3492)
+
+   > Adds a Release Date field for mobile first run experiments to accurately calculate enrollment and durations to save multiple manual calculations. (erichards)
+
+🍏🤖 [Supporting Mobile Teams](https://mozilla-hub.atlassian.net/browse/EXP-3231)
+
+   > Ongoing support for mobile engineers using Nimbus. (jhugman)
+
+🍏🤖 [Core Active Behavioral Targeting](https://mozilla-hub.atlassian.net/browse/EXP-2600)
+
+   > Applications are now able to share information with Nimbus necessary for the categorization of users as “core-active” and experiments/messaging are able to target based on this categorization. (chumphreys)
+
+## Desktop
+
+💻 [Desktop Localization](https://mozilla-hub.atlassian.net/browse/EXP-3164)
+
+   > Enables localized experiments on the Nimbus Desktop client. (brennie)
+
+## Jetstream & Results Analysis
+
+📈 [Improve Results Links](https://mozilla-hub.atlassian.net/browse/EXP-3145)
+
+   > Experimenter links to Results page are now disabled for experiments that have no results, and the Results page redirects to the Summary and shows appropriate messages to the user when there are no results. (mikew)
+
+📈 [Shared Schema for Jetstream Results](https://mozilla-hub.atlassian.net/browse/EXP-3423)
+
+   > Define and create a new shared schema for consumption by Experimenter and Jetstream, ingestion of analysis results, and export of analysis results from Jetstream. (mikew)
+
+## Cirrus
+
+☁️ [Cirrus API Core Behavior](https://mozilla-hub.atlassian.net/browse/EXP-3469)
+
+   > Implements the core behaviour of the Cirrus API such that it receives a request from the client and can respond with the features and feature values. This requires fetching active experiments from Remote Setting, using the Nimbus SDK to compute enrollment, and using the FML to evaluate feature configurations. (ykhurana)
+
+☁️  [Nimbus FML for Cirrus](https://mozilla-hub.atlassian.net/browse/EXP-3454)
+
+   > Creates FML client and Python bindings for Cirrus. (chumphreys)
+
+☁️  [Nimbus SDK for Cirrus](https://mozilla-hub.atlassian.net/browse/EXP-3158)
+
+   > Given the existing Nimbus SDK library, this work creates an extensible, shared Nimbus library in which the majority of the functions are static as opposed to the stateful functions within the existing library. (chumphreys)
+
+☁️  [Cirrus Documentation](https://mozilla-hub.atlassian.net/browse/EXP-3121)
+
+   > Document/create ADRs for requirements, implementations, and system design diagrams. (ykhurana)
+
+## Experimenter
+
+🙌 [Outreachy contributions](https://mozilla-hub.atlassian.net/browse/EXP-3318)
+
+   > Tasks that were completed by Outreachy intern applicants. (ykhurana)
```

Comment:
Could we add the following blurb about the changelog Avishkar worked on: 

With the added capacity from this  internship, we improved the experimenter experience by adding a changelog page for each experimenter.  The primary goal is to introduce a new page that leverages Django, Tailwind CSS, and Alpine.js (new pattern!).